src/main/java/core/basesyntax/Operations.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public enum Operations {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r"),
+    NOT_FOUND("Not found");
```

Comment:
interesting architecture decision, but what's the use of operation that you can't even apply? I think it's better to terminate the program immediately if we encounter unrecognized operation, file is probably corrupted